DICOMwebBrowser/DICOMwebBrowser.py

```diff
@@ -943,9 +947,16 @@ def populateSeriesTableWidget(self, studyUID, series):
     rowIndex = self.seriesTableRowCount
     table.setRowCount(rowIndex + len(series))
 
-    from dicomweb_client.api import load_json_dataset
+    
+    import dicomweb_client 
+    from packaging import version 
     for serieJson in series:
-      serie = load_json_dataset(serieJson)
+      
+      if (version.parse(dicomweb_client.__version__) < version.parse("0.54")):
+        from dicomweb_client.api import load_json_dataset
```

Comment:
Could you refactor the code so that `load_json_dataset` function is set once - where `dicomweb_client` is imported - and not for each series?

Reply:
I made a note at https://github.com/pieper/SlicerDICOMwebBrowser/pull/1#discussion_r797089551 which details a potential method for avoiding this split in code pathways.

Reply:
Thank you @jamesobutler your fix is good (we can just use `pydicom.dataset.Dataset.from_json`).
@pieper @deepakri201 @fedorov could you make that change and give it a quick test?

Reply:
I can make this change and test it.